CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - [v1.3.0-alpha.3](#v130-alpha3)
   - [Downloads](#downloads)
+- [v1.3.0-alpha.3](#v130-alpha3)
```

Comment:
That file was updated by hand for the last alpha release and there are some extra lines that the TOC generator is taking into account.  I can send a separate PR or you can just remove the second occurrences of these lines in the file:

# v1.3.0-alpha.3

[Documentation](http://kubernetes.github.io) & [Examples](http://releases.k8s.io/master/examples)